src/main/java/gg/solarmc/clans/config/MessageConfig.java

```diff
@@ -17,6 +17,7 @@
 
     @ConfKey("clanNotExist")
     @ConfComments("When a clan is not found or it doesn't exist")
+    @ConfDefault.DefaultString("Clan does not exist!")
```

Comment:
Write a unit test which loads the configuration that will ensure the defaults are set properly.